lib/Context/Internal/ProcessHub.php

```diff
@@ -74,8 +74,11 @@ public function __construct()
         $this->watcher = Loop::onReadable(
             $this->server,
             static function (string $watcher, $server) use (&$keys, &$acceptor): void {
-                // Error reporting suppressed since stream_socket_accept() emits E_WARNING on client accept failure.
-                while ($client = @\stream_socket_accept($server, 0)) {  // Timeout of 0 to be non-blocking.
+                $read = [$server];
+                $empty = [];
+                // To prevent debuggers from triggering when stream_socket_accept() emits E_WARNING on client accept failure,
+                // first check to see if there is a stream waiting for us.
+                while (stream_select($read, $empty, $empty, 0) && $client = \stream_socket_accept($server, 0)) {  // Timeout of 0 to be non-blocking.
```

Comment:
As this is a hard dependency on stream_select, it will usually break with file descriptors > 1024, which is the default limit for stream_select. I think a custom error handler might be better if we want to avoid the warning.